build.gradle.kts

```diff
@@ -93,7 +96,7 @@ tasks.test {
     finalizedBy("jacocoTestReport")
     doLast {
         println("View code coverage at:")
-        println("file://$buildDir/reports/jacoco/test/html/index.html")
+        println("file://$buildDir/reports/coverage/index.html")
```

Comment:
You might want to undo this change and rebase/merge from `master` so you get the fix